### ISSUE TITLE
Don´t change password master

### ISSUE THREAD

**Issue by vtkageyama** [nuxsmin/sysPass#1721]
sysPass version	3.2 (322.21031301)
Config: 322.21031301
App: 322.21031301
DB: 322.21031301
Database	SERVER_VERSION : 5.5.5-10.5.11-MariaDB
CLIENT_VERSION : mysqlnd 7.4.21
SERVER_INFO : Uptime: 98638 Threads: 3 Questions: 9884 Slow queries: 0 Opens: 48 Open tables: 41 Queries per second avg: 0.100
CONNECTION_STATUS : Localhost via UNIX socket
Nome: syspass@localhost
PHP	Version: 7.4.21
Extensions: Core, date, libxml, openssl, pcre, zlib, filter, hash, Reflection, SPL, session, standard, apache2handler, bz2, calendar, ctype, curl, dom, mbstring, fileinfo, ftp, gd, gettext, iconv, intl, json, ldap, exif, mysqlnd, PDO, Phar, SimpleXML, sockets, sodium, sqlite3, tokenizer, xml, xmlwriter, xsl, mysqli, pdo_mysql, pdo_sqlite, xmlreader, zip, Zend OPcache
Unavailable extensions:
Used Memory: 2048 KB
User: apache
Download rate: 43 MB/s
OP Cache
num_cached_scripts : 468
num_cached_keys : 880
max_cached_keys : 16229
hits : 160876
start_time : 1625149677
last_restart_time : 0
oom_restarts : 0
hash_restarts : 0
manual_restarts : 0
misses : 468
blacklist_misses : 0
blacklist_miss_ratio : 0
opcache_hit_rate : 99,709936533122

**Describe the bug**
I try change password master but after confirm the change I try login again and password continue the same

I check the table config and the value don't change.

**Comment by nuxsmin** [comment]
Hello,

how do you proceed to change the password?

Regards.

**Comment by s-praveen-kumar** [comment]
Am facing similar issue. I set Maintenance mode. Updated the Master Password by entering the Old and New and confirm.
Then I hit Enter and ACCEPT the popup. Then I Press the button to update Hash.

Expectation : I am prompted with master password at next login. And things work well.

Actual Observation : I am prompted with master password at login. Entering new password shows wrong. Old password still works.
